packages/e2e-tests/prebundle-svelte-deps/vite.config.js

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from 'vite';
+import { svelte } from '@sveltejs/vite-plugin-svelte';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+	plugins: [svelte()],
+	optimizeDeps: {
+		// TODO this must be excluded because nested has an scss dep that prebundle can't handle!
+		// figure out how to exclude it automatically or at least tell the user about it in a more friendly way
+		exclude: ['e2e-test-dep-svelte-nested'],
+		include: [
+			// TODO without this, it fails for module.exports in the browser.
+			//   shouldn't prebundling take care of it?
+			'e2e-test-dep-svelte-hybrid > e2e-test-dep-cjs-only'
```

Comment:
Looks like this will be fixed by https://github.com/sveltejs/vite-plugin-svelte/pull/482. The requests shows that it's not using the prebundled files.

Reply:
Here's a snippet of it
```js
import Hybrid from '/@fs/Users/bjorn/Work/oss/vite-plugin-svelte/node_modules/.pnpm/file+packages+e2e-tests+_test_dependencies+svelte-hybrid/node_modules/e2e-test-dep-svelte-hybrid/index.js';
import Simple from '/@fs/Users/bjorn/Work/oss/vite-plugin-svelte/node_modules/.pnpm/file+packages+e2e-tests+_test_dependencies+svelte-simple/node_modules/e2e-test-dep-svelte-simple/index.js';
import { Message as Nested } from '/@fs/Users/bjorn/Work/oss/vite-plugin-svelte/node_modules/.pnpm/file+packages+e2e-tests+_test_dependencies+svelte-nested/node_modules/e2e-test-dep-svelte-nested/src/index.js';
import { setSomeContext } from '/node_modules/.vite/deps/e2e-test-dep-svelte-api-only.js?v=89f48f1a';
import { getContext } from '/node_modules/.vite/deps/svelte.js?v=89f48f1a';
const file = "src/App.svelte";
```